l10n_us_form_1099/__manifest__.py

```diff
@@ -6,15 +6,22 @@
     "version": "12.0.1.0.0",
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "summary": "Add 1099 field to res.partner that will auto-check supplier",
+    "summary": """Add 1099 field to res.partner that will auto-check supplier
+                  and aid in reporting payments to these 1099 partners for US
+                  tax purposes""",
```

Comment:
@brian10048 I believe this will be too long for the summary. Let's go with `Manage 1009 types and suppliers`